keps/sig-network/4444-service-routing-preference/README.md

```diff
@@ -405,6 +364,23 @@ NOTE: The fact that EndpointSlice hints are not expected to be implemented by
   kube-controller-manager based implementations to say that we do support the
   standard heuristics.
 
+### Changes within kube-proxy
+
+**Present behaviour:** kube-proxy only considers EndpointSlice hints for route
+programming if the `service.kubernetes.io/topology-aware-hints` annotation is
+set to "Auto"
+
+**New behaviour:** Irrespective of what the annotation
+`service.kubernetes.io/topology-aware-hints` or field `routingPreference` are
+set to (or even if they are not set at all), kube-proxy will always consider
+EndpointSlice hints.
```

Comment:
"... kube-proxy will always consider EndpointSlice hints (assuming this feature gate is enabled)"

Reply:
Added!